README.md

```diff
@@ -68,9 +68,11 @@ Options can include:
 }
 ```
 
-#### `await blobs.clear(id)`
+#### `await blobs.clear(id, opts)`
 Remove a blob from the core.
 
+`opts` are the same as `Hypercore.clear` method.
```

Comment:
Shouldn't the reference be the other way around, if we want to make it implicit? Since Hypercore will be calling Blobs rather than the other way around. But probably easier to just mention the one opt here